asab/task.py

```
@@ -49,8 +49,8 @@ async def finalize(self, app):
 				await task
 			except asyncio.CancelledError:
 				pass
-			except Exception as e:
-				L.exception("Error '{}' during task service:".format(e))
+			except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)